packages/openid4vc-client/src/OpenId4VcClientService.ts

```diff
@@ -232,4 +294,89 @@ export class OpenId4VcClientService {
 
     return storedCredential
   }
+
+  public async requestCredential(agentContext: AgentContext, options: RequestCredentialOptions) {
+    const credentialFormat = 'ldp_vc'
+    const client = await OpenID4VCIClient.initiateFromURI({
+      issuanceInitiationURI: options.issuerUri,
+      kid: options.kid,
+      flowType: AuthzFlowType.AUTHORIZATION_CODE_FLOW,
+      alg: Alg.EdDSA,
+    })
+
+    const accessToken = await client.acquireAccessToken({
+      clientId: options.clientId,
+      code: options.code,
+      codeVerifier: options.codeVerifier,
+      redirectUri: options.redirectUri,
+    })
+
+    const serverMetadata = await client.retrieveServerMetadata()
+    // @ts-ignore
```

Comment:
We should avoid ts ignore as much as possible. I think it would be better to add a cast one time and cast it to the right type once as this disables all type checks for yhay line.

Same for the scope having the wrong type